scss/custom/_modal.scss

```diff
@@ -25,19 +22,14 @@
   .modal-header,
   .modal-footer {
     // Fixed for older phones
-    display: -webkit-flex; /* stylelint-disable-line */
-    display: -moz-box; /* stylelint-disable-line */
-    display: -ms-flexbox; /* stylelint-disable-line */
+    display: flex;
 
     width: 100%;
     box-shadow: $modal-full-shadow;
   }
 
   .modal-body {
     // Fixed for older phones
-    -webkit-box-flex: 1;   /* stylelint-disable-line */ /* OLD - iOS 6-, Safari 3.1-6 */
-    -webkit-flex: 1;       /* stylelint-disable-line */ /* Safari 6.1+. iOS 7.1+, BB10 */
-    -ms-flex: 1;           /* stylelint-disable-line */ /* IE 10 */
     flex: 1;               /* NEW, Spec - Firefox, Chrome, Opera */
```

Comment:
You can get rid of the comment "NEW ..." too, doesn't really matter if it's new spec or not.